optimize/util/optimize-commons/src/main/java/io/camunda/optimize/dto/optimize/query/entity/EntityNameRequestDto.java

```diff
@@ -27,7 +27,4 @@ public class EntityNameRequestDto {
 
   @QueryParam("reportId")
   private String reportId;
-
-  @QueryParam("eventBasedProcessId")
-  private String eventBasedProcessId;
```

Comment:
This is a separate breaking API change, has it been documented and handled by the FE?

Reply:
It has been documented in the pr description, my preference is to first get BE approved and then have FE add the required changes to this branch, then have FE approve those changes